packages/gatsby-source-medium/src/gatsby-node.js

```diff
@@ -115,3 +110,507 @@ exports.sourceNodes = async (
+      postCount: Int
+      coverImage: MediumPostVirtualsTagsMetadataCoverImage
+    }
+
+    type MediumPostVirtualsTagsMetadataCoverImage {
+      id: String
+      originalWidth: Int
+      originalHeight: Int
+      isFeatured: Boolean
+      unsplashPhotoId: String
+      focusPercentX: Int
+      focusPercentY: Int
+      alt: String
+      backgroundSize: String
+      filter: String
+      externalSrc: String
+      repairedAt: Date @dateformat
+    }
+
+    type MediumPostVirtualsLinks @derivedTypes {
+      entries: [MediumPostVirtualsLinksEntries]
+      version: String
+      generatedAt: Date @dateformat
+    }
+
+    type MediumPostVirtualsLinksEntries @derivedTypes {
+      url: String
+      alts: [MediumPostVirtualsLinksEntriesAlts]
+      httpStatus: Int
+    }
+
+    type MediumPostVirtualsLinksEntriesAlts {
+      type: Int
+      url: String
+    }
+
+    type MediumPostVirtualsTopics @derivedTypes {
+      topicId: String
+      slug: String
+      createdAt: Date @dateformat
+      deletedAt: Date @dateformat
+      image: MediumPostVirtualsTopicsImage
+      name: String
+      description: String
+      visibility: Int
+      type: String
+    }
+
+    type MediumPostVirtualsTopicsImage {
+      id: String
+      originalWidth: Int
+      originalHeight: Int
+    }
+
+    type MediumPostPreviewContent @derivedTypes {
+      bodyModel: MediumPostPreviewContentBodyModel
+      isFullContent: Boolean
+      subtitle: String
+    }
+
+    type MediumPostPreviewContentBodyModel @derivedTypes {
+      paragraphs: [MediumPostPreviewContentBodyModelParagraphs]
+      sections: [MediumPostPreviewContentBodyModelSections]
+    }
+
+    type MediumPostPreviewContentBodyModelParagraphs @derivedTypes {
+      name: String
+      type: Int
+      text: String
+      layout: Int
+      metadata: MediumPostPreviewContentBodyModelParagraphsMetadata
+      markups: [MediumPostPreviewContentBodyModelParagraphsMarkups]
+      alignment: Int
+      hasDropCap: Boolean
+    }
+
+    type MediumPostPreviewContentBodyModelParagraphsMetadata {
+      id: String
+      originalWidth: Int
+      originalHeight: Int
+      isFeatured: Boolean
+      unsplashPhotoId: String
+      alt: String
+    }
+
+    type MediumPostPreviewContentBodyModelParagraphsMarkups {
+      type: Int
+      start: Int
+      end: Int
+      href: String
+      title: String
+      rel: String
+      anchorType: Int
+      userId: String
+    }
+
+    type MediumPostPreviewContentBodyModelSections {
+      startIndex: Int
+    }
+
+    type MediumPostPreviewContent2 @derivedTypes {
+      bodyModel: MediumPostPreviewContent2BodyModel
+      isFullContent: Boolean
+      subtitle: String
+    }
+
+    type MediumPostPreviewContent2BodyModel @derivedTypes {
+      paragraphs: [mediumPostPreviewContent2BodyModelParagraphs]
+      sections: [mediumPostPreviewContent2BodyModelSections]
+    }
+
+    type mediumPostPreviewContent2BodyModelParagraphs @derivedTypes {
```

Comment:
Curious why is it a lowercase? Probably shouldn't be.

Reply:
I looked at another related PR where you advised to lower-case type names, but then I realized it's not needed here. Thanks for pointing that out!